pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -349,6 +349,15 @@ type TiKVSpec struct {
 	// +optional
 	StorageClassName *string `json:"storageClassName,omitempty"`
 
+	// Subdirectory within the volume to store TiKV Data. By default, the data
+	// is stored in the root directory of volume which is mounted at
+	// /var/lib/tikv.
+	// Specifying this will change the data directory to a subdirectory, e.g.
+	// /var/lib/tikv/data if you set the value to "data".
+	// Defaults to "" (volume's root).
+	// +optional
+	DataSubDir string `json:"dataSubDir,omitempty"`
```

Comment:
```suggestion
	DataSubDir *string `json:"dataSubDir,omitempty"`
```

Reply:
not necessary, as the empty string is the default value

Reply:
we don't need to distinguish empty value from nil value here